zookeeper-server/src/main/java/org/apache/zookeeper/client/StaticHostProvider.java

```diff
@@ -331,7 +331,7 @@ private InetSocketAddress nextHostInReconfigMode() {
         return null;
     }
 
-    public InetSocketAddress next(long spinDelay) {
+    public InetSocketAddress next(long delay) {
```

Comment:
IIUC, `serverAddresses` is a bit like Circular Queue, when spinDelay>0 and serverAddresses list is too short(e.g size=1), to avoid connecting that server quickly again, it will sleep `spinDelay`.That's why the word:`spin` used

Reply:
OK. I have reverted the change. Please check again.